images/deprecating.md

```diff
@@ -5,13 +5,13 @@ description: Learn how to deprecate an image.
 
 You can mark images as deprecated to prevent them from being used to create new environments.
 
-1. Go to "Images" and find the image you want to deprecate.
-2. Click "Edit".
-3. Select the "Deprecate this image" checkbox.
-4. Click "Update Image" to save your changes.
+1. Go to **Images** and find the image you want to deprecate.
```

Comment:
I'm not sure if this is better or worse, but: want to deprecate -> mark as deprecated?